static/css/main.css

```diff
@@ -44,6 +44,10 @@ ul {
   width: 40px;
 }
 
+.twitter  {
+  text-align:center;
+}
```

Comment:
Lets go with 100% width instead of centering

Try the selector `.twitter > iframe`